index.html

```diff
@@ -0,0 +1,164 @@
+
+<body>
+    <!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <div class="dark-nav">
+                <ul>
+                    <li><a href="https://www.facebook.com/"><img src="images/facebook.png" alt="fcebook icon"></a></li>
+                    <li><a href="https://twitter.com/home"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
+                </ul>
+            </div>
+
+            <nav class="navbar">
+                <a href="index.html" class="nav-branding"><img src="images/dark logo.png" alt="logo"></a>
+
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link orange-text">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#program" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#featured" class="nav-link">Featured</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#sponsor" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link border-text">Community</a>
+                    </li>
+                </ul>
+
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+        <!--HERO-->
+        <section class="hero">
+            <p class="quote">
+                “Welcome to the javascript society summit”
+            </p>
+            <h1 class="first-heading">
+                Javascript society<br>
+                Summit<br>
+                2023
+            </h1>
+            <div class="first-para">
+                <p>
+                    An Javascript coders community that believes in the value of openness, sharing, creativity and tech
+                    empowerment. Creating a positive change with people all over the world. We are hosting Africa
+                    biggest open source conference this November, in Nairobi, Kenya.
+                </p>
+            </div>
+            <p class="second-heading">
+                2023.07.3(MON) ~ 7(FRI)
+            </p>
+            <p class="second-para">
+                @ UON Towers, Nairobi, Kenya and more.
+            </p>
+        </section>
+    </div>
+    <!--MAIN PROGRAM-->
+    <section id="program" class="main-program">
+        <div class="main-heading">
+            <h2>Main Program</h2>
+            <hr class="main-line">
+        </div>
+        <div class="program-cards">
```

Comment:
- [ ] The desktop version of your project still has content overlapping. Please add margin between the two sections. 😺 

**DESIGN**
![des1](https://github.com/peter-mwovi/Capstone-project/assets/99656687/a6830d30-fefa-48d9-813f-796e96b0e098)

**YOUR IMPLEMENTATION**
![des2](https://github.com/peter-mwovi/Capstone-project/assets/99656687/8f591531-f1a8-4755-bb3f-31b5c713d890)

Reply:
@wayungi Please help me, because I cannot replicate the overlaping issue  on my end. Everything seems okay.

![image](https://github.com/peter-mwovi/Capstone-project/assets/134924276/6a768bab-0a2b-48ec-95d1-e063cc0f43b0)